gorush/TAG

```diff
@@ -1 +1 @@
-1.11.2.1
+1.11.2-beta.1
```

Comment:
[The semantic versioning](https://semver.org/) defines that any pre-release (versions having `-xxx`)
is smaller than the version w/o pre-release.  So, `1.11.2-beta.1` is smaller than `1.11.2`.

I don't think this is your intention.

Reply:
@ymmt2005 

I see. I'll change this to 1.11.3-beta.
Thank you.

Reply:
@emiksk 
I'd recommend to use `emiksk-1.11.2.1` or like that.

Reply:
@ymmt2005 
> `emiksk-1.11.2.1`
That sounds good!
I'll update it.

Reply:
Done